### PR TITLE
COMMERCE-9363 Update CPTaxCategory edit external reference page

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9363

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce

**Comment by pyoo47** [comment]
ci:test:commerce

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-tax-category-web/src/main/java/com/liferay/commerce/product/tax/category/web/internal/portlet/action/EditCPTaxCategoryMVCActionCommand.java [hunk 1]
```diff
@@ -86,63 +86,73 @@ else if (exception instanceof CPTaxCategoryNameException ||
...
-					"/cp_tax_category/edit_cp_tax_category");
+				if (cmd.equals(Constants.ADD)) {
+					actionResponse.setRenderParameter(
+						"mvcRenderCommandName",
+						"/cp_tax_category/add_cp_tax_category");
+				}
+				else {
+					actionResponse.setRenderParameter(
+						"mvcRenderCommandName",
+						"/cp_tax_category/edit_cp_tax_category");
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-tax-category-web/src/main/java/com/liferay/commerce/product/tax/category/web/internal/portlet/action/EditCPTaxCategoryMVCActionCommand.java:97]: @luca-pellizzon I had to add a if statement here.

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce

**Review by luca-pellizzon — APPROVED** [review]

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:commerce

**Comment by dsitu** [comment]
ci:reevaluate:1297401_4628

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
sent to #3037

**Comment by dsitu** [comment]
Closing, resent here https://github.com/liferay-commerce/liferay-portal/pull/3037

**Pull request closed without merging**